src/multimanifestmodule.ts

```diff
@@ -8,21 +8,23 @@ import { stackanalysismodule } from './stackanalysismodule';
 import { authextension } from './authextension';
 import { StatusMessages } from './statusMessages';
 import { DependencyReportPanel } from './dependencyReportPanel';
+import { Apiendpoint } from './apiendpoint';
 
 export module multimanifestmodule {
   export const form_manifests_payload = (resultPath): any => {
     return new Promise((resolve, reject) => {
       manifestFileRead(resultPath)
         .then(item => {
           let form_data = {
-            'manifest[]': [],
-            'filePath[]': [],
-            'license[]': [],
-            origin: 'lsp'
+            'manifest': '',
+            'file_path': '',
+            'license[]': '',
+            origin: 'lsp',
```

Comment:
There is no origin field in the form data. It should be removed.
```suggestion
```

Reply:
fixed